packages/logs/src/domain/logsCollection/console/consoleCollection.ts

```diff
@@ -35,7 +35,7 @@ export function startConsoleCollection(configuration: LogsConfiguration, lifeCyc
               }
             : undefined,
         status: LogStatusForApi[log.api],
-      },
+      } satisfies RawConsoleLogsEvent,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** It seems that the type is inferred thanks to the `origin` attribute . Is it worth having the `satisfies RawConsoleLogsEvent`?‏